src/main/kotlin/io/github/mojira/risa/infrastructure/Jira.kt

```diff
@@ -55,7 +55,7 @@ fun getTicketsForSnapshot(jiraClient: JiraClient, config: Config, currentSnapsho
 
 @SuppressWarnings("MaxLineLength")
 private fun getJql(currentSnapshot: Snapshot): String =
-    "project = MC AND affectedVersion = \"${currentSnapshot.name}\" AND created > ${currentSnapshot.releasedDate.minus(1L, ChronoUnit.DAYS).toEpochMilli()} AND (status = Open OR status = Reopened OR resolution in (\"Works As Intended\", \"Fixed\", \"Awaiting Response\", \"Unresolved\", \"Won't Fix\")) ORDER BY \"Confirmation Status\" DESC, key ASC"
+    "project = MC AND affectedVersion = \"${currentSnapshot.name}\" AND created > ${currentSnapshot.releasedDate.minus(1L, ChronoUnit.DAYS).toEpochMilli()} AND (status = Open OR status = Reopened OR resolution in (\"Works As Intended\", \"Fixed\", \"Awaiting Response\", \"Unresolved\", \"Won't Fix\")) AND ((resolution != \"Awaiting Response\" OR resolution is EMPTY ) OR (resolution = \"Awaiting Response\" AND updated < 24h)) ORDER BY \"Confirmation Status\" DESC, key ASC"
```

Comment:
```suggestion
    "project = MC AND affectedVersion = \"${currentSnapshot.name}\" AND created > ${currentSnapshot.releasedDate.minus(1L, ChronoUnit.DAYS).toEpochMilli()} AND (status = Open OR status = Reopened OR resolution in (\"Works As Intended\", \"Fixed\", \"Awaiting Response\", \"Unresolved\", \"Won't Fix\")) AND ((resolution != \"Awaiting Response\" OR resolution is EMPTY ) OR (resolution = \"Awaiting Response\" AND updated > -24h)) ORDER BY \"Confirmation Status\" DESC, key ASC"
```
Otherwise it will just return all tickets that it did before.

Also this JQL is a bit messy with all the escaping, is there no way to like use raw strings here or something?

Reply:
I think we can use """ but it looks a bit worse